app/run.py

```diff
@@ -73,113 +73,115 @@ def generate_parser(parser=None):
     """
     if not parser:
         parser = argparse.ArgumentParser(
-            prog='dcan-pipelines',
+            prog='abcd-hcp-pipeline',
             description=__doc__,
             formatter_class=argparse.RawDescriptionHelpFormatter,
-            epilog=__references__
+            epilog=__references__,
+            usage='%(prog)s bids_dir output_dir --freesurfer-license=<LICENSE>'
         )
     parser.add_argument(
         'bids_dir',
-        help='path to the input bids dataset root directory.  Read more '
-             'about bids format in the link in the description.  It is '
-             'recommended to use the dcan bids gui or dcm2bids to convert '
-             'from participant dicoms to bids.'
+        help='Path to the input BIDS dataset root directory.  Read more '
+             'about the BIDS standard in the link in the description.  It is '
+             'recommended to use Dcm2Bids to convert from participant dicoms '
+             'into BIDS format.'
     )
     parser.add_argument(
         'output_dir',
-        help='path to the output directory for all intermediate and output '
-             'files from the pipeline, also path in which logs are stored.'
+        help='Path to the output directory for all intermediate and output '
+             'files from the pipeline, which is also where logs are stored.'
     )
     parser.add_argument(
         '--version', '-v', action='version', version='%(prog)s ' + __version__
     )
     parser.add_argument(
         '--participant-label', dest='subject_list', metavar='ID', nargs='+',
-        help='optional list of participant ids to run. Default is all ids '
-             'found under the bids input directory.  A participant label '
-             'does not include "sub-"'
+        help='Optional list of participant IDs to run. Default is all IDs '
+             'found under the BIDS input directory. The participant label '
+             'does not include the "sub-" prefix'
     )
     parser.add_argument(
         '--freesurfer-license', dest='freesurfer_license', 
         metavar='LICENSE_FILE',
-        help='if using docker or singularity, you will need to acquire and '
-             'provide your own freesurfer license. The license can be '
+        help='If using docker or singularity, you will need to acquire and '
+             'provide your own FreeSurfer license. The license can be '
              'acquired by filling out this form: '
              'https://surfer.nmr.mgh.harvard.edu/registration.html'
     )
     parser.add_argument(
         '--all-sessions', dest='collect', action='store_true',
-        help='collapses all sessions into one when running a subject.'
+        help='Collapses all sessions into one when running a subject.'
```

Comment:
Where is the data required for this? Should there be 1 fmap dir or does every session need one, etc?